src/api.ts

```diff
@@ -92,23 +113,43 @@ router.post('/logout', (req, res) => {
 router.get('/allRoles', getAllRoles);
 
 // Members
-router.get('/allMembers', allMembers);
-router.post('/setMember', setMember);
-router.post('/deleteMember', deleteMember);
+router.get('/allMembers', async (req, res) => {
+  let handled = await allMembers(req, res);
+  res.status(handled.status).json(handled);
```

Comment:
what does this line do?

Reply:
@morganbelous That line just passes the request and response to the `allMembers` handler, and then when that returns, it returns an object like `{ status: number, members: [...], ...}`. So, we just grab the status out of the response object and set the response to that status number. Then, on the frontend, we can check the status number to log out or do whatever else we need to.